app/controllers/api/v1/moves_controller.rb

```diff
@@ -73,6 +74,25 @@ def find_move
           .includes(:from_location, :to_location, person: { profiles: %i[gender ethnicity] })
           .find(params[:id])
       end
+
+      def import_moves_from_nomis
```

Comment:
It's worth considering wrapping this in a `begin...rescue` block. If Elite2 is unavailable this would block the whole request I think. We probably want to show the cached moves still, in this case.

Reply:
It makes sense to return cached data if it's available rather than failing the request. I just wonder whether we need some way to warn the user that the data they are seeing is out of date. This might be best tackled when implementing a cache expiry function (later).